app/src/main/java/com/gbbeard/countingbutton/MainActivity.java

```diff
@@ -0,0 +1,72 @@
+package com.gbbeard.countingbutton;
+
```

Comment:
The `win`, `loss` and `tie` buttons seem to work as expected (so technically you have completed the lab), but I'm not sure what the `games played` button is supposed to do.  If it's supposed to sum the other three values, that's not working correctly; but it does increment the `games total` value.  It's a bit confusing.  :)